### PR TITLE
[RHPAM-2310] Add jboss-kie-module for process migration service

### PR DESCRIPTION
**Pull request by ruromero**
Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

Fix https://issues.jboss.org/browse/RHPAM-2310

Adds jboss-kie-module for process migration service including scripts and default configurations to be used in the resulting image.

Allows configuring extra classpath entries (that would be provided inside the image) and additional configuration file to override defaults.

### CODE REVIEW

File: jboss-kie-process-migration/added/configuration/application-roles.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+kermit=admin,user
+gonzo=user
```
Comment (errantepiphany) [inline jboss-kie-process-migration/added/configuration/application-roles.properties]: Why include harcoded properties? Perhaps these would be better set as env vars, and then have the jboss-kie-process-migration.sh script dynamically create the users and roles properties files?

File: jboss-kie-process-migration/added/configuration/application-users.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+kermit=thefrog
+gonzo=thegreat
```
Comment (errantepiphany) [inline jboss-kie-process-migration/added/configuration/application-users.properties]: Same hardcoding question as above.

File: jboss-kie-process-migration/added/configuration/application-roles.properties (hunk 1 above)
Comment (spolti) [reply jboss-kie-process-migration/added/configuration/application-roles.properties]: +1 otherwise lgtm.

**Comment by ruromero** [comment]
@spolti @errantepiphany please review the new approach. Thanks

**Comment by spolti** [comment]
@ruromero looks like your branch does n ot have the circleci configuration file, do you mind to rebase your branch?

**Comment by ruromero** [comment]
@spolti done

**Comment by ruromero** [comment]
@spolti done!

**Comment by errantepiphany** [comment]
@ruromero Please squash the commits?

**Pull request merged**